stable/yourls/templates/deployment.yaml

```
@@ -0,0 +1,167 @@
+        {{- toYaml .Values.podAnnotations | nindent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.podAnnotations }}
+        {{- toYaml .Values.metrics.podAnnotations | nindent 8 }}
+      {{- end }}
+      {{- end }}
+    spec:
+      {{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      hostAliases:
+        - ip: "127.0.0.1"
+          hostnames:
+            - "status.localhost"
+      containers:
+        - name: "{{ template "yourls.fullname" . }}"
+          image: {{ template "yourls.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
+          env:
+            - name: YOURLS_DB_HOST
+            {{- if .Values.mariadb.enabled }}
+              value: {{ template "mariadb.fullname" . }}:{{ .Values.mariadb.service.port }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.host | quote }}
+            {{- end }}
+            - name: YOURLS_DB_NAME
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.name | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.database | quote }}
+            {{- end }}
+            - name: YOURLS_DB_USER
+            {{- if .Values.mariadb.enabled }}
+              value: {{ .Values.mariadb.db.user | quote }}
+            {{- else }}
+              value: {{ .Values.externalDatabase.user | quote }}
+            {{- end }}
+            - name: YOURLS_DB_PASS
+              valueFrom:
+                secretKeyRef:
+                {{- if .Values.mariadb.enabled }}
+                  name: {{ template "mariadb.fullname" . }}
+                  key: mariadb-password
+                {{- else }}
+                  name: {{ printf "%s-%s" .Release.Name "externaldb" }}
+                  key: db-password
+                {{- end }}
+            - name: YOURLS_DB_PREFIX
+              value: {{ .Values.yourlsTablePrefix | quote }}
+            - name: YOURLS_SITE
+              value: {{ .Values.yourlsSite | quote }}
+            - name: YOURLS_USER
+              value: {{ .Values.yourlsUsername | quote }}
+            - name: YOURLS_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "yourls.fullname" . }}
+                  key: yourls-password
+          {{- if .Values.extraEnv }}
+            {{- toYaml .Values.extraEnv | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: 80
+            - name: https
+              containerPort: 443
+          livenessProbe:
+            httpGet:
+              path: /admin/
+            {{- if not .Values.healthcheckHttps }}
+              port: http
+            {{- else }}
+              port: https
+              scheme: HTTPS
+            {{- end }}
+          {{- toYaml .Values.livenessProbe | nindent 10 }}
```

@LeoColomb I tested it on our environment, and it seems the indent is wrong for liveness and readiness probes :)

```suggestion
            {{- toYaml .Values.livenessProbe | nindent 12 }}
```